### ISSUE TITLE
--css and --js flags broken

### ISSUE THREAD

**Issue by reiner-dolp** [jbt/docker#98]
Using the `--css` and/or `--js` options results in the following exception  in nodejs v4.4.5:

```
path.js:341
    throw new TypeError('ext must be a string');
    ^

TypeError: ext must be a string
    at win32.basename (path.js:341:11)
    at Array.map (native)
    at null.<anonymous> (node_modules\docker\src\docker.js:1338:26)
```
I had to substitute all lines containing `map(path.basename)` with 
```js
map(function(file) {
    return path.basename(file);
})
```

**Comment by jbt** [comment]
I think this is the same as in #82 and should be fixed by #86 which (because I'm a terrible person, sorry!) has been neglected _again_, argh! I'll get that PR merged and 1.0 finalised this weekend.

**Comment by reiner-dolp** [comment]
You're right. That's an exact duplicate. Missed that one while scanning existing issues and pull requests.
